HW0/P5/.ipynb_checkpoints/P5-checkpoint.ipynb

```diff
@@ -0,0 +1,115 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x1065ea290>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(bag_count,single_worker,'r-',label='1 worker')\n",
+    "plt.plot(bag_count,infinite_worker,'b-',label='Infinite workers')\n",
+    "plt.xlabel('Number of Bags (N)')\n",
+    "plt.ylabel('Time (Seconds)')\n",
+    "plt.legend()\n",
+    "savefig('P5.png', bbox_inches='tight')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 2",
+   "language": "python",
+   "name": "python2"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 0
+}
```

Comment:
This is for the iPython notebook I used to create the plot. It was just saving a checkpoint of my files. Nothing you really need to have.